internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -241,6 +241,13 @@ func SetRLimit(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFun
 // DropCapabilities drops some capabilities so that they can't be restored by child processes.
 func DropCapabilities(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFunc, string) {
 	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) error {
+		prop, err := kernel.ReadParam(&kernel.Param{Key: "kernel.kexec_load_disabled"})
```

Comment:
I think `DropCapabilities` is called in `Initialize` sequence before we load the config.
How would it be possible to set this kernel param before this method? Or I'm missing some other way to set up kernel params? :thinking:

Reply:
I tried to explain that in the changelog.

Yes, that is weird :)

Reply:
ah, install arguments. Yes, now that makes sense.

Reply:
Kernel params are driven by:

* for images (including ISO) - whatever is in the image
* for PXE boot - PXE kernel args
* for installed Talos booting from disk via `machine.install.extraKernelArgs`

Technically we don't have to do anything for install phase most probably, but rather for the phase when Talos boots normally.

Reply:
If we get docs for `kexec` we may want to include that release notes block there.